cypress/pages/modals/message-box.ts

```diff
@@ -0,0 +1,18 @@
+import { BasePage } from "../base";
+
+export class MessageBox extends BasePage {
+	getters = {
+		modal: () => cy.get('.el-message-box', { withinSubject: null }),
```

Comment:
could not find a non-hacky way to add test ids to element ui components

Reply:
I suppose we'll have to compromise on this one for `element-ui`.